exercises/practice/bob/.approaches/answer-array/content.md

```diff
@@ -0,0 +1,41 @@
+# Answer array
+
+```csharp
+    // uses System.Linq;
+    private static readonly string[] answers = {"Whatever.", "Sure.", "Whoa, chill out!", "Calm down, I know what I'm doing!"};
+    
+    public static string Response(string statement)
+    {
+        var input = statement.TrimEnd();
+        if (input == "") return "Fine. Be that way!";
+        
+        var isShout = (input.Any(c => char.IsLetter(c)) && input.ToUpper() == input) ? 2: 0;
+        
+        var isQuestion = (input.EndsWith('?')) ? 1: 0;
+        
+        return answers[isShout + isQuestion];
+    }
```

Comment:
Yes, I agree that being able to copy/paste them and play with them would be really good, so including all the necessary using etc statements makes sense.

Reply:
I also agree to have the whole code. I had that originally, but took it out because of a comment made during the last conference call about not putting in all of the class stuff, but now I think that was only meant for the snippet,